CONTRIBUTING.md

```diff
@@ -36,7 +36,8 @@ supported branch of PHP that the bug affects (only green branches on
 [the supported version page](https://www.php.net/supported-versions.php) are
 supported). For example, at the time of writing, the lowest supported version is
 PHP 8.0, which corresponds to the `PHP-8.0` branch in Git. Please also make sure
-you add a link to the PR in the bug on [the bug tracker](https://bugs.php.net/).
+you add a link to the PR in the bug on [the bug tracker](https://github.com/php/php-src/issues)
+or [the old bug tracker](https://bugs.php.net/).
```

Comment:
GitHub will automatically link the PR in the issue if it contains `Closes/fixes GH-xyz` so I wonder if this is necessary for GitHub issues.

Reply:
I think it is still helpful to see which (open) tickets already have a PR (and vice versa).